dataStores/router.js

```diff
@@ -0,0 +1,12 @@
+const ChatController = require('./chat/ChatController');
+const chatController = new ChatController();
+const router = require('express').Router();
+
+router.get('/messages', chatController.getMessages);
+router.get('/users', chatController.getUsers);
+
+router.post('/messages', chatController.saveMessage);
+router.post('/connect', chatController.connect);
+router.post('/disconnect', chatController.disconnect);
```

Comment:
A to nie usunięcie?

Reply:
jak chcemy być purystami to powinno być post /user i delete /user

Reply:
Umawialiśmy się na spotkaniu, że używamy tylko metod `GET` oraz `POST`

Reply:
chyba nie, śmiało możesz popłynąć z metodami będzie ciekawiej ;)

Reply:
@macraq imo wtedy powinniśmy też wchodzić w bardziej rozbudowane statusy niż `200`/`400`. @wreszelewski @nivlheim @J-a-r-e-k  wypowiedzcie sie tez